packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -31,6 +31,7 @@ import * as errors from '../universal/errors'
 import {isRemote} from '@salesforce/pwa-kit-runtime/utils/ssr-server'
 import {proxyConfigs} from '@salesforce/pwa-kit-runtime/utils/ssr-shared'
 import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
+import logger from '@salesforce/pwa-kit-runtime/utils/logger'
```

Comment:
Are we using this logger? I saw an import but never seen it get called

Reply:
We aren't directly using the logger in the file. The import is there to ensure that the console overrides apply to any client-side and server-side code using the `console` methods, while keeping the logic in one place, in this case the `pwa-kit-runtime`

I wasn't sure about where and how we want to implement the override. I'm open to suggestions on the best practices for doing this.

Reply:
@adamraya you might want to use the syntax `import 'module-name'` if there is no expectation in having an exported value. 

You might also want to not export anything from your logger util if it's only overriding the console globally.

Alternatively you can export an initializer that you call once that will do the reassignment of console.

Reply:
Thanks for the suggestion @bendvc. I made the change.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/44f4553e6c59c5a70ff2053107a860c5ad49d8f5/packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js#L34